docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  epochtal:
+    image: ghcr.io/p2r3/epochtal:latest
+    ports:
+      - 8080:8080/tcp
+    # Use environment values from the .env file.
+    # If you don't want this, comment the following line and use the individual environment variables below instead.
+    env_file: '.env'
+    #environment:
+    #  STEAM_API_KEY:
+    #  DISCORD_API_KEY:
+    #  JWT_SECRET:
+    #  INTERNAL_SECRET:
+    #  DISCORD_CHANNEL_ANNOUNCE:
+    #  DISCORD_CHANNEL_REPORT:
+    #  DISCORD_CHANNEL_UPDATE:
+    #  WEB_URL: # This is optional
+    volumes:
+      # You can bind the data directory of epochtal to your host (recommended).
+      # Comment out the following line if you don't want that.
+      - ./data:/app/data
+      # You can also bind the 'secrets' directory to your host. This contains SSL certificates (if used) and curation weights
+      #- ./secrets:/app/secrets
```

Comment:
Should mention this in the README.md

Reply:
As in adding it to the "plain docker" command?

I can probably do that, but the README as I've made it now is more of a quickstart guide, and i feel like it should just contain the minimum needed for a deployment - most users probably dont need to bind this. It makes more sense to me that this is documented by itself in the place where it is declared.

Reply:
You decide, but I personally would like it in the README.md for completeness-sake